src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num) {
+      calculator.result += num;
```

Comment:
replace calculator with this everywhere

Reply:
when i did it the tests fail

Reply:
I can't understand why

`function makeCalculator() {
  const calculator = {
    result: 0,

    add(num) {
      this.result += num;

      return calculator;
    },

    subtract(num) {
      this.result -= num;

      return calculator;
    },

    multiply(num) {
      this.result *= num;

      return calculator;
    },

    divide(num) {
      this.result /= num;

      return calculator;
    },

    reset() {
      this.result = 0;

      return calculator;
    },

    operate(callback, num) {
      callback(num);

      return calculator;
    },
  };

  return calculator;
}`

Reply:
I did it, but I still don't understand why a simple replacement broke everything.